reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md

```diff
@@ -0,0 +1,652 @@
+---
+title: What's New in PowerShell 7.1
+description: New features and changes released in PowerShell 7.1
+ms.date: 11/10/2020
+---
+
+# What's New in PowerShell 7.1
+
+TODO: Refresh
+
+PowerShell 7.1 is an open-source, cross-platform (Windows, macOS, and Linux) edition of PowerShell,
+built to manage heterogeneous environments and hybrid cloud.
+
+In this release, we're introducing a number of new features, including:
+
+## Where can I install PowerShell?
+
+TODO: Refresh
+
+PowerShell 7 currently supports the following operating systems on x64, including:
+
+- Windows 8.1, and 10
+- Windows Server 2012, 2012 R2, 2016, and 2019
+- macOS 10.13+
+- Red Hat Enterprise Linux (RHEL) / CentOS 7
+- Fedora 30+
+- Debian 9
+- Ubuntu LTS 16.04+
+- Alpine Linux 3.8+
```

Comment:
Ubuntu 16.10, 17.04, and 17.10 are out of support. There could be other examples of this too.

Rather than list versions here, just list OS names and link to the support lifecycle article: /powershell/scripting/powershell-support-lifecycle